lib/modules/manager/nuget/util.ts

```diff
@@ -52,13 +52,22 @@ export async function getConfiguredRegistries(
   }
 
   const packageSources = nuGetConfig.childNamed('packageSources');
+  const disabledPackageSources = nuGetConfig.childNamed(
+    'disabledPackageSources',
+  );
```

Comment:
Move this lower closer to where it's used

Reply:
👍 Done